### PR TITLE
docs: Cli windows installation

### PR DESCRIPTION
**Pull request by kostis-codefresh**
Ongoing work for https://github.com/argoproj/argo-cd/issues/5635

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [X] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [X] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [] Does this PR require documentation updates?
* [X] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [X] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

**Review by terrytangyuan — APPROVED** [review]
LGTM. Thanks!

File: docs/getting_started/install.md [hunk 1]
```diff
@@ -29,10 +37,20 @@ kubectl -n argocd apply -f https://raw.githubusercontent.com/argoproj/argo-cd/st
...
+When you are ready to use Argo CD in a production environment you have
+several installation options.
+
+For a production setup you should pay attention to extra aspects of your installation such as:
+
+* defining the correct resource limits
+* applying proper security constraints
+* setting up high availability
+* managing Argo CD upgrades
+* monitoring Argo CD itself
```
Comment (jannfis) [inline docs/getting_started/install.md:53]: Going forward, we should probably fill this with links to the respective sections in the docs

File: docs/getting_started/install.md [hunk 2]
```diff
@@ -220,7 +238,15 @@ Also, please note that this is not an officially supported installation method.
 Please direct all questions or problems you face using the Helm chart directly
 to the chart's maintainers.
 
+## Alternative installation methods
+
+You can also an infrastructure tool such as [Terraform](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs), [Pulumi](https://www.pulumi.com/registry/packages/kubernetes/) and [Crossplane](https://github.com/crossplane-contrib/provider-kubernetes) to manage your Argo CD installation.
```
Comment (jannfis) [inline docs/getting_started/install.md:243]:
```suggestion
You can also use infrastructure tools such as [Terraform](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs), [Pulumi](https://www.pulumi.com/registry/packages/kubernetes/) or [Crossplane](https://github.com/crossplane-contrib/provider-kubernetes) to manage your Argo CD installation.
```

**Review by jannfis — APPROVED** [review]
LGTM, thanks @kostis-codefresh. Just one minor nitpick w/ suggestion, please see below.

File: docs/getting_started/install.md (hunk 1 above)
Comment (kostis-codefresh) [reply docs/getting_started/install.md:53]: Agreed

**Pull request merged**